### ISSUE TITLE
UniqueTogetherValidator error message should return verbose_name 

### ISSUE THREAD

**Issue by FingerLiu** [tomchristie/django-rest-framework#4134]
## Checklist

- [x] I have verified that that issue exists against the `master` branch of Django REST framework.
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] This is not a usage question. (Those should be directed to the [discussion group](https://groups.google.com/forum/#!forum/django-rest-framework) instead.)
- [x] This cannot be dealt with as a third party library. (We prefer new functionality to be [in the form of third party libraries](http://www.django-rest-framework.org/topics/third-party-resources/#about-third-party-packages) where possible.)
- [x] I have reduced the issue to the simplest possible case.
- [ ] I have included a failing test as a pull request. (If you are unable to do so we can still accept the issue.)

## Steps to reproduce
1. def model with unique_together attr
``` python
class Category(models.Model):
     name = models.CharField(max_length=20, verbose_name=_('name'))
     parent = models.ForeignKey('self', related_name='children', verbose_name=_('name'))
     class Meta:
           unique_together = ('name', 'parent',)
```
and create ModelViewSet for the model
2. create category(name='a', parent=1) 
3. call create request with data (name='a', parent=1)
## Expected behavior
Since I have set `USE_I18N = True` and `USE_L10N = True`, and added locale/xxx/django.po correctly.
I expect a error message well translated.
## Actual behavior
fields with verbose_name are not translated.
As below
![image](https://cloud.githubusercontent.com/assets/3981064/15415524/ba1d840a-1e75-11e6-9049-951ee94a8713.png)

I've inspsected the DRF code, there are not any logic dealing with verbose_name in **UniqueTogetherValidator**.
Currently it just get field name of serializer field and return it to error_message of UniqueTogetherValidator.
As blow
https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/validators.py#L149

Is this by purpose or by mistake?
I can try to make a  PR  with test case and fix if it is needed.

**Comment by FingerLiu** [comment]
????

**Comment by tomchristie** [comment]
> ????

@FingerLiu We will get around to reviewing this. Right now we're only on volunteer-time, and there's plenty of other issues that need reviewing too, so we won't always be able to get to an issue immediately.

Give us a bit more time.

**Comment by tomchristie** [comment]
Haven't attempted to replicate, but appears to be a valid issue from the description.
Would be interesting to know if this is only an issue for the translated case, or it it's an issue always.
Also if anyone is able to replicate and confirm please do pitch in.

**Comment by FingerLiu** [comment]
@tomchristie this is only an issue for the translated case

**Comment by tomchristie** [comment]
Believe this to be closed by #3435
